src/main/java/teammates/storage/entity/Comment.java

```diff
@@ -98,6 +106,8 @@ public Comment(String courseId, String giverEmail, CommentRecipientType recipien
         this.showRecipientNameTo = showRecipientNameTo;
         this.createdAt = date;
         this.commentText = comment;
+        this.lastEditorEmail = (lastEditorEmail == null) ? giverEmail : lastEditorEmail;
+        this.lastEditedAt = (lastEditedAt == null) ? date : lastEditedAt;
```

Comment:
It's okay to have null at entity level (eg Comment/FeedbackResponseComment), because every legacy comment will have null values for lastEditorEmail & date.

instead, null value should be replaced by default value at attribute level (eg CommentAttribute/FeedbackResponseCommentAttribute)